web/src/test/java/it/pagopa/selfcare/mscore/web/controller/InstitutionControllerTest.java

```diff
@@ -1073,6 +1079,87 @@ void findFromProduct() throws Exception {
         verifyNoMoreInteractions(institutionService);
     }
 
+    /**
+     * Method under test: {@link InstitutionController#getInstitutionBrokers(String, InstitutionType)}
+     */
+    @Test
+    void getInstitutionBrokers() throws Exception {
+        // Given
+        final String productId = "test";
+        final InstitutionType type = InstitutionType.PT;
+        Institution institution = new Institution();
+        institution.setId("id");
+
+        // When
+        when(institutionService.getInstitutionBrokers(any(), any())).thenReturn(List.of(institution));
+        MockHttpServletRequestBuilder requestBuilder = MockMvcRequestBuilders.get(BASE_URL + "/{productId}/brokers/{institutionType}", productId, type);
+        ResultActions actualPerformResult = MockMvcBuilders.standaloneSetup(institutionController)
+                .setCustomArgumentResolvers(new PageableHandlerMethodArgumentResolver())
+                .build()
+                .perform(requestBuilder);
+        actualPerformResult
+                .andExpect(MockMvcResultMatchers.status().isOk())
+                .andExpect(MockMvcResultMatchers.content().contentType("application/json"));
+        // Then
+        verify(institutionService, times(1))
+                .getInstitutionBrokers(productId, type);
+        verifyNoMoreInteractions(institutionService);
+    }
+
+    /**
+     * Method under test: {@link InstitutionController#getInstitutionBrokers(String, InstitutionType)}
+     */
+    @Test
+    void getInstitutionBrokers2() throws Exception {
```

Comment:
it might be useful to give the test method a more meaningful name....what is the different with getInstitutionBroker() ?